packages/consumption/test/modules/requests/itemProcessors/formField/FormFieldRequestItemProcessor.test.ts

```diff
@@ -0,0 +1,1024 @@
+                    message: `The response provided cannot be used to accept the form field.`
+                });
+            });
+
+            test("returns an error when it is tried to accept a string form field with a free string whose length is smaller than the min", function () {
+                const anEmptyString = "";
+
+                const requestItem = FormFieldRequestItem.from({
+                    mustBeAccepted: true,
+                    title: "aFormField",
+                    settings: StringFormFieldSettings.from({ min: aMin })
+                });
+
+                const acceptParams: AcceptFormFieldRequestItemParametersJSON = {
+                    accept: true,
+                    response: anEmptyString
+                };
+
+                const result = processor.canAccept(requestItem, acceptParams);
+
+                expect(result).errorValidationResult({
+                    code: "error.consumption.requests.invalidAcceptParameters",
+                    message: `The length of the response cannot be smaller than the min.`
+                });
+            });
+
+            test("returns an error when it is tried to accept a string form field with a free string whose length is greater than the max", function () {
+                const aStringTooLong = "aStringTooLong";
+
+                const requestItem = FormFieldRequestItem.from({
+                    mustBeAccepted: true,
+                    title: "aFormField",
+                    settings: StringFormFieldSettings.from({ max: aMax })
+                });
+
+                const acceptParams: AcceptFormFieldRequestItemParametersJSON = {
+                    accept: true,
+                    response: aStringTooLong
+                };
+
+                const result = processor.canAccept(requestItem, acceptParams);
+
+                expect(result).errorValidationResult({
+                    code: "error.consumption.requests.invalidAcceptParameters",
+                    message: `The length of the response cannot be greater than the max.`
+                });
+            });
+        });
+
+        describe("IntegerFormFieldSettings", function () {
+            test("can accept an integer form field with a free integer", function () {
+                const requestItem = FormFieldRequestItem.from({
+                    mustBeAccepted: true,
+                    title: "aFormField",
+                    settings: IntegerFormFieldSettings.from({})
+                });
+
+                const acceptParams: AcceptFormFieldRequestItemParametersJSON = {
+                    accept: true,
+                    response: anInteger
+                };
+
+                const result = processor.canAccept(requestItem, acceptParams);
+
+                expect(result).successfulValidationResult();
+            });
+
+            test("returns an error when it is tried to accept an integer form field with no free integer", function () {
```

Comment:
> with no free integer

please just write "with a double"

In general I'm not a fan of the wording `free x` can we get rid of that?